sources/osgUtil/DisplayNodeGraphVisitor.js

```diff
@@ -130,7 +130,11 @@ define( [
                 }
             }
 
+            if ( node.getSourceGeometry ) node.addChild( node.getSourceGeometry() );
+
             this.traverse( node );
+
```

Comment:
It's temporary.
It's just one way to see if we have a morph or not when using the debug node graph tool.